### PR TITLE
WIP Create disability rating calculator

### PR DESCRIPTION
**Pull request by aubreyarcangel**
## Description
Content PR - https://github.com/department-of-veterans-affairs/vagov-content/pull/442
First QA build of Disability Calculator which will be accessible at `/disability/about-disability-ratings-beta/`.  Additional work will be continued in a new PR.

## Testing done
Tested locally

## Acceptance criteria
- [ ] First build of disability calculator is working and ready for initial QA

## Definition of done
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

**Comment by ncksllvn** [comment]
Hey @aubreyarcangel , just as an FYI, there are a considerable amount on [JS-Lint errors](http://jenkins.vetsgov-internal/blue/organizations/jenkins/testing%2Fvets-website/detail/disabiliy-rating-calculator-18214/10/pipeline) here. Some of them can be ignored if they're a natural part of WIP code, but the bulk of them might be easier to fix now rather than later. LMK if you're having issues with your SOCKS proxy.

You can address these by exploring our [lint commands](https://github.com/department-of-veterans-affairs/vets-website/blob/master/package.json#L16). You can also set up your editor to lint automatically. Just search the plugin store for your editor.

File: src/applications/facility-locator/components/ResultsList.jsx [hunk 1]
```diff
@@ -110,8 +110,8 @@ class ResultsList extends Component {
                 <SearchResult result={r} />
               </div>
             ) : (
-              <SearchResult key={r.id} result={r} />
-            );
+                <SearchResult key={r.id} result={r} />
+              );
             /* eslint-enable prettier/prettier */
           })}
         </div>
```
Comment (ncksllvn) [inline src/applications/facility-locator/components/ResultsList.jsx]: @aubreyarcangel, I think you forgot to switch back to the master branch when you were switching to this [pull request](https://github.com/department-of-veterans-affairs/vets-website/pull/10161). This commit should be reverted.

File: src/applications/disability-calculator/components/DisabilityRatingCalculator.jsx [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+
+  handleChange = (e, idx) => {
+    const curRatings = this.state.ratings;
+    curRatings[idx][e.target.name] =
+      // eslint-disable-next-line radix
+      e.target.name === 'rating' ? parseInt(e.target.value) : e.target.value;
+    this.setState({ ratings: curRatings }, () => {
+      console.log(this.state);
+    });
+  };
```
Comment (ncksllvn) [inline src/applications/disability-calculator/components/DisabilityRatingCalculator.jsx]: I'm not sure if this `handleChange ` works or not, but this feels very frail. I think we can address this later though after getting the add/delete/clear handlers cleaned up.

**Review by ncksllvn — COMMENTED** [review]
Coming along!

**Review by narin — APPROVED** [review]
SME wants to see the current progress on staging.  We've updated the route to be `/disability/about-disability-ratings-beta/` and made some of the recommended fixes.  Will merge this PR so SME can start QA on staging, and create a new PR to handle the remainder of @ncksllvn's recommendations.

**Pull request merged**